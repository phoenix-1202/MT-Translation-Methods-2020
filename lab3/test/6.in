funThatShowDifferenceBetweenArgsAndFunctionsWithoutArgs :: Int -> Int -> Int
funThatShowDifferenceBetweenArgsAndFunctionsWithoutArgs x y = (x) + (((y _)))