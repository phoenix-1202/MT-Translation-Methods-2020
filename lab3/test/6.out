#include <bits/stdc++.h>;
using namespace std;

int funThatShowDifferenceBetweenArgsAndFunctionsWithoutArgs(int arg1, int arg2) {
return (arg1 + y());
}

//EXIT